step.yml

```diff
@@ -14,6 +14,12 @@ toolkit:
   go:
     package_name: github.com/bitrise-steplib/bitrise-step-save-cache
 
+deps:
+  brew:
+  - name: zstd
+  apt_get:
+  - name: zstd
```

Comment:
My instinct would be to use a built-in compression binary for compressing files. I don't know much about `zstd` though, could you explain why you chose to use this over something built-in?

Reply:
I made some benchmarks of the different compression tools and options, see here: https://bitrise.atlassian.net/wiki/spaces/ENGI/pages/2351169600/RFC+New+key-based+caching+steps#Compression-benchmarks

The speedup is so significant that it's basically a free win

Reply:
Yeah, I agree with all of this! Until then, nightly E2E tests will also catch a potential error like that.